### PR TITLE
Hide data(Censor sensitive information of customers)

### PR DESCRIPTION
**Pull request by Kelly9373**
For issue: https://github.com/CS2103-AY1819S1-F10-2/main/issues/61

### CODE REVIEW

File: src/main/java/seedu/address/model/loan/Phone.java [hunk 1]
```diff
@@ -27,4 +27,11 @@ public static boolean isValidPhone(String objString) {
         return objString.matches("\\d{3,}");
     }
 
+    @Override
+    public String getCensored() {
+        String output = this.value;
+        output = output.charAt(0) + "xxxxx" + output.substring(output.length() - 2);
+        return output;
```
[the remark this answers is not in the document]
Comment (wn96) [reply src/main/java/seedu/address/model/loan/Phone.java]: https://beginnersbook.com/2013/12/java-string-indexof-method-example/

File: src/main/java/seedu/address/model/loan/Email.java [hunk 2]
```diff
@@ -42,4 +42,20 @@ public static boolean isValidEmail(String objString) {
             + DOMAIN_FIRST_CHARACTER_REGEX + DOMAIN_MIDDLE_REGEX + DOMAIN_LAST_CHARACTER_REGEX);
     }
 
+    @Override
+    public String getCensored() {
```
[the remark this answers is not in the document]
Comment (Kelly9373) [reply src/main/java/seedu/address/model/loan/Email.java]: Fixed :)

File: src/main/resources/view/LoanListCard.fxml [hunk 3]
```diff
@@ -7,30 +7,84 @@
...
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="On " wrappingWidth="27.21875" />
+            <Label fx:id="time" styleClass="cell_small_label" text="\$time" />
+            </children>
+         </TextFlow>
+         <TextFlow prefHeight="17.0" prefWidth="130.0">
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="With rate: " />
+            <Label fx:id="rate" prefHeight="17.0" prefWidth="33.0" styleClass="cell_small_label" text="\$rate" />
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="/hr" />
```
Comment (wn96) [inline src/main/resources/view/LoanListCard.fxml]: Hard coding this seems really bad. If the rate is 1.2345678, you might get text over your `/hr` text.

Comment (wn96) [reply src/main/resources/view/LoanListCard.fxml]: If you are unable to resolve this, can we set rate to only accept 2 decimal place?

Comment (Kelly9373) [reply src/main/resources/view/LoanListCard.fxml]: The rate is alr limited to 2 decimal places.

Comment (wn96) [reply src/main/resources/view/LoanListCard.fxml]: Roger

**Pull request merged**